README.md

```diff
@@ -7,24 +7,12 @@
 ---
 
 <p align=center>
-  <a href="https://reactos.org/project-news/reactos-0414-released/">
-    <img alt="ReactOS 0.4.14 Release" src="https://img.shields.io/badge/release-0.4.14-0688CB.svg">
-  </a>
-  <a href="https://reactos.org/download/">
-    <img alt="Download ReactOS" src="https://img.shields.io/badge/download-latest-0688CB.svg">
-  </a>
-  <a href="https://sourceforge.net/projects/reactos/">
-    <img alt="SourceForge Download" src="https://img.shields.io/sourceforge/dm/reactos.svg?colorB=0688CB">
-  </a>
-  <a href="https://github.com/reactos/reactos/blob/master/COPYING">
-    <img alt="License" src="https://img.shields.io/badge/license-GNU_GPL_2.0-0688CB.svg">
-  </a>
-  <a href="https://reactos.org/donate/">
-    <img alt="Donate" src="https://img.shields.io/badge/%24-donate-E44E4A.svg">
-  </a>
-  <a href="https://twitter.com/reactos">
-    <img alt="Follow on Twitter" src="https://img.shields.io/twitter/follow/reactos.svg?style=social&label=Follow%20%40reactos">
-  </a>
+  <a href="https://reactos.org/project-news/reactos-0414-released/"><img alt="ReactOS 0.4.14 Release" src="https://img.shields.io/badge/release-0.4.14-0688CB.svg"></a>
+  <a href="https://reactos.org/download/"                          ><img alt="Download ReactOS"       src="https://img.shields.io/badge/download-latest-0688CB.svg"></a>
+  <a href="https://sourceforge.net/projects/reactos/"              ><img alt="SourceForge Download"   src="https://img.shields.io/sourceforge/dm/reactos.svg?colorB=0688CB"></a>
+  <a href="https://github.com/reactos/reactos/blob/master/COPYING" ><img alt="License"                src="https://img.shields.io/badge/license-GNU_GPL_2.0-0688CB.svg"></a>
+  <a href="https://reactos.org/donate/"                            ><img alt="Donate"                 src="https://img.shields.io/badge/%24-donate-E44E4A.svg"></a>
+  <a href="https://twitter.com/reactos"                            ><img alt="Follow on Twitter"      src="https://img.shields.io/twitter/follow/reactos.svg?style=social&label=Follow%20%40reactos"></a>
```

Comment:
Hmh I would suggest removing all these space paddings.

Reply:
Double that.

Reply:
ok
changed